dump.rdb

```diff
@@ -0,0 +1 @@
+REDIS0006�ܳC�Z��V
```

Comment:
Can you remove this file, and maybe add a .gitignore entry if you think this might happen again?